samples/sample_common/src/plugin_utils.cpp

```diff
@@ -142,8 +142,12 @@ const mfxPluginUID & msdkGetPluginUID(mfxIMPL impl, msdkComponentType type, mfxU
         case MSDK_VENCODE:
             switch(uCodecid)
             {
+// On Android implementaion of HEVC encoder is placed in libmfx
+// That's why we don't need default plugins for this codec
+#if !defined(ANDROID)
```

Comment:
Isn't that the case now for non-Android as well? I thought this plugin is just a stub. In this case, why differentiate?

Reply:
@dvrogozh , the reason of difference - different build configuration: for android build no target of default plugin, but for another system it still exist.
But generally you are right, if run HW HEVC encode on linux without plugin - should works OK.

Reply:
by the way same actual for line 129

Reply:
That's a question what we want to demonstrate in our samples. If we consider start switching users to work with hw codecs right from the library (which makes sense since it avoids separate path to load plugins), we better provide common code for all platforms.